src/webots/nodes/WbElevationGrid.cpp

```diff
@@ -46,12 +46,13 @@ void WbElevationGrid::init() {
   mData = NULL;
   mMinHeight = 0;
   mMaxHeight = 0;
+  mIs90DegreesRotated = true;
```

Comment:
Why is it rotated?

Reply:
It seems to be justified by a difference between ODE and VRML in https://github.com/cyberbotics/webots/blob/feature-enu-geometries/src/webots/nodes/WbGeometry.hpp#L171
However, given that we are leaving the VRML world with Y vertical, we could also afford to adopt the ODE standard for elevation grids, remove this rotation and adapt our documentation and examples accordingly.